### PR TITLE
change a test

### PR DESCRIPTION
**Pull request by levynir**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -160,8 +160,8 @@ describe('Tests mongoose-timezone-offset', function() {
     });
 
     describe('edge cases', function() {
-        it('Should not break with bad offset of 17', async function() {
-            const now = moment.utc().utcOffset(17);
```
Comment (levynir) [inline test.js:164]: this is a good idea 

---






**Sema Reaction:** :hammer_and_wrench: Fix | **Sema Tags:** None

File: test.js [hunk 2]
```diff
@@ -160,8 +160,8 @@ describe('Tests mongoose-timezone-offset', function() {
...
+        it('Should not break with bad offset of 14', async function() {
+            const now = moment.utc().utcOffset(14);
```
Comment (levynir) [inline test.js:164]: 🏆 

---



**Sema Reaction:** :trophy: Awesome | **Sema Tags:** None

File: test.js [hunk 3]
```diff
@@ -160,8 +160,8 @@ describe('Tests mongoose-timezone-offset', function() {
     });
 
     describe('edge cases', function() {
-        it('Should not break with bad offset of 17', async function() {
```
Comment (levynir) [inline test.js:163]: this is not so good because you did not return a promise
see 
<p>An <code>async</code> function always wraps the return value in a <code>Promise</code>. Using <code>return await</code> is therefore redundant.</p>
<h2>Noncompliant Code Example</h2>
<pre>
async function foo() {
  // ...
}

async function bar() {
  // ...
  return await foo(); // Noncompliant
}
</pre>
<h2>Compliant Solution</h2>
<pre>
async function foo() {
  // ...
}

async function bar() {
  // ...
  return foo();
}
</pre>



---
**Sema Reaction:** :ok_hand: Looks good | **Sema Tags:** None

Comment (levynir) [inline test.js:163]: ffgff

---
**Sema Reaction:** :ok_hand: Looks good | **Sema Tags:** Secure, Inefficient, Inelegant, Not reusable, Fault-tolerant, Maintainable
